examples/gaussian_process/utils.py

```diff
@@ -0,0 +1,79 @@
+'''
+Gaussian Process utils. Design inspired by GPflow.
+'''
+
+import numpy as np
+import tensorflow as tf
+import zhusuan as zs
+
+
+class RBFKernel:
+
+    def __init__(self, n_inp, name='rbf_kernel', dtype=tf.float32):
```

Comment:
doc_string since the name of `n_inp` is uninformative.

Reply:
what about n_covariates?